java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -705,6 +707,338 @@ public static CompletableFuture<Long> arrlen(
+     *     If the result is out of the range of 64-bit IEEE double, an error is raised.
+     * </ul>
+     * @example
+     *     <pre>{@code
+     * Json.set(client, "doc", "$", "{\"a\": [], \"b\": [1], \"c\": [1, 2], \"d\": [1, 2, 3]}").get();
+     * var res = Json.nummultby(client, gs("doc"), gs("$.d[*]"), 2.0).get();
+     * assert res.equals(gs("[2,4,6]")); // Multiplies each element in the `d` array by 2.
+     * res = Json.nummultby(client, gs("doc"), gs(".c[1]"), 2.0).get();
+     * assert res.equals(gs("12")); // Multiplies the second element in the `c` array by 2.
+     * }</pre>
+     */
+    public static CompletableFuture<GlideString> nummultby(
+        @NonNull BaseClient client,
+        @NonNull GlideString key,
+        @NonNull GlideString path,
+        double number) {
+        return executeCommand(
+            client,
+            new GlideString[] {gs(JSON_NUMMULTBY), key, path, gs(Double.toString(number))}
+        );
+    }
+
+    /**
+     * Multiplies the JSON value(s) at the specified <code>path</code> by <code>number</code> within the JSON document stored at <code>key</code>.
+     *
+     * @param client The client to execute the command.
+     * @param key The key of the JSON document.
+     * @param path The path within the JSON document.
+     * @param number The number to multiply by.
+     * @return <ul>
+     *     <li>For JSONPath (<code>path</code> starts with <code>$</code>):<br>
+     *         Returns a bytes string representation of an array of bulk strings, indicating the new values after multiplication for each matched <code>path</code>.<br>
+     *         If a value is not a number, its corresponding return value will be <code>null</code>.<br>
+     *         If <code>path</code> doesn't exist, a byte string representation of an empty array will be returned.
+     *     </li>
+     *     <li>For legacy path (<code>path</code> doesn't start with <code>$</code>):<br>
+     *         Returns a bytes string representation of the resulting value after multiplication.<br>
+     *         If multiple paths match, the result of the last updated value is returned.<br>
+     *         If the value at the <code>path</code> is not a number or <code>path</code> doesn't exist, an error is raised.
+     *     </li>
+     *     If <code>key</code> does not exist, an error is raised.<br>
+     *     If the result is out of the range of 64-bit IEEE double, an error is raised.
+     * </ul>
+     * @example
+     *     <pre>{@code
+     * Json.set(client, "doc", "$", "{\"a\": [], \"b\": [1], \"c\": [1, 2], \"d\": [1, 2, 3]}").get();
+     * var res = Json.nummultby(client, "doc", "$.d[*]", 2).get();
+     * assert res.equals("[2,4,6]"); // Multiplies each element in the `d` array by 2.
+     * res = Json.nummultby(client, "doc", ".c[1]", 2).get();
+     * assert res.equals("12"); // Multiplies the second element in the `c` array by 2.
+     * }</pre>
+     */
+    public static CompletableFuture<String> nummultby(
```

Comment:
```suggestion
    public static CompletableFuture<Object> nummultby(
```
??? your tests seem to indicate that this returns an array

Reply:
Man, I can't tell you why. I sat down with James and Yi-Pin to debug my tests because they were failing when run locally and we had to change them to compare Object[] instead. Then I check CI and it's completely different.